lib/components/Buttons/styles.ts

```diff
@@ -16,13 +16,10 @@ export const button_primary = css`
   transition: opacity 0.3s ease 0s;
   background-color: var(--klima-green);
 
-  &:hover:not(:disabled),
-  &:focus:not(:disabled) {
+  &:active:not(:disabled) {
+    transform: scale(0.95);
     opacity: 0.6;
   }
-  &:focus:not(:disabled) {
-    transform: scale(0.9);
-  }
   &,
```

Comment:
This was causing buttons to stay scaled down on user click.

Reply:
Nice one, yeah I noticed that too. What do you think utilising `:active` so the button can return to its original size after a user click?